lib/simplelog/simplelog.go

```diff
@@ -0,0 +1,71 @@
+// Package simplelog contains a very simple interface for logging strings at either Debug, Info, or Error levels
+package simplelog
+
+import (
+	"context"
+
+	"oss.terrastruct.com/d2/lib/log"
+	"oss.terrastruct.com/util-go/cmdlog"
+)
+
+type Logger interface {
+	Debug(string)
+	Info(string)
+	Error(string)
+}
+
+type logger struct {
+	logDebug *func(string)
+	logInfo  *func(string)
+	logError *func(string)
+}
+
+func (l logger) Debug(s string) {
+	if l.logDebug != nil {
+		(*l.logDebug)(s)
+	}
+}
+func (l logger) Info(s string) {
+	if l.logInfo != nil {
+		(*l.logInfo)(s)
+	}
+}
+func (l logger) Error(s string) {
+	if l.logError != nil {
+		(*l.logError)(s)
+	}
+}
+
+func Make(logDebug, logInfo, logError *func(string)) Logger {
```

Comment:
doesn't need to be public

Reply:
hmm if we want to be able to use `imgbundler.BundleLocal(context.Context, simplelog.Logger, []byte, bool)` we want this to be public or it is limited to using `FromLibLog` or `FromCmdLog` or you have to re-implement the `simplelog.Logger` interface